README.md

```diff
@@ -27,60 +74,106 @@ mutation SignInMutation($input: signInInput!) {
+
+### GraphQL `signIn` mutation & JWT tokens
 
-### With a JWT token
+A better way than cookies to stay authenticated is to rely on [JWT tokens](https://jwt.io/).
 
-To authenticate by JWT you need to request a valid JWT token in the `signIn` mutation :
+We provide JWT tokens within the `signIn` mutation. They can be retrieve using the following mutation:
 
 ```gql
-  mutation SignInMutation($input: signInInput!) {
-    signIn(input: $input) {
-      currentUser {
-        slug
-        jwtToken(aud: "YOUR_AUD") {
-          token
-          expiredAt
-        }
+mutation SignInMutation($input: signInInput!) {
+  signIn(input: $input) {
+    currentUser {
+      slug
+      jwtToken(aud: "<YourAud>") {
+        token
+        expiredAt
       }
-      ...
     }
   }
+}
 ```
 
-`YOUR_AUD` is a mandatory string parameter that helps identifying your app. Also, tokens generated with an `aud` are scoped to this `aud`. We recommend to use an `aud` reflecting the name of your app - like `myrealappname` - to make it easier to debug & track.
+`<YourAud>` is a mandatory _string_ parameter that helps (us & you) identifying your app. We recommend to use an `aud` reflecting the name of your app - like `myappname` - to make it easier to debug & track.
 
-You shall then pass it to all next requests through the following headers:
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'x-csrf-token: <YourCSRFTokenValue>' \
+-H 'cookie: _sorare_session_id=<YourCookieValue>' \
+-d '{
+  "operationName": "SignInMutation",
+  "variables": { "input": { "email": "<YourEmail>", "password": "<YourHashPassword>" } },
+  "query": "mutation SignInMutation($input: signInInput!) { signIn(input: $input) { currentUser { slug email jwtToken(aud: \"<YourAud>\") { token expiredAt } } errors { message } } }"
+}'
+
+{"data":{"signIn":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>","jwtToken":{"token":"<YOUR JWT TOKEN>","expiredAt":"..."}},"errors":[]}}}
 ```
-Authorization: Bearer YOUR_TOKEN
-JWT-AUD: YOUR_AUD
+
+You shall then pass the token with an `Authorization` header alongside a `JWT-AUD` header to all next API requests instead of the `cookie` and the `x-csrf-token` headers:
+
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'Authorization: Bearer <YourJWTToken>' \
+-H 'JWT-AUD: <YourAud>' \
+-d '{
+    "operationName": "CurrentUserQuery",
+    "query": "query CurrentUserQuery { currentUser { slug email } }"
+}'
+
+{"data":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>"}}}
 ```
 
 ## 2FA
 
 For account with 2FA enabled the `signIn` mutation will return an `otpSessionChallenge` instead of the `currentUser`.
 
-You then need to do a second call to the `signIn` mutation providing only the `otpSessionChallenge` and a valid `otpAttempt` :
+You then need to do a second call to the `signIn` mutation providing only the `otpSessionChallenge` and a valid `otpAttempt` as input:
```

Comment:
as _input_:
-> as _inputs_:

Reply:
or something like:

In this case, you will need to make another call to the `signIn` mutation and provide otpSessionChallenge` value you received and a one-time token from your 2FA device as `otpAttempt`: